examples/get-my-contract-ids-flow/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Looks like newer versions of `prettier` lowercase doctype and there's nothing we can do about it.